app/views/shared/_admin_flash.html.erb

```diff
@@ -1,8 +1,12 @@
-<div class="container">
+<div class="container" data-controller="flash">
```

Comment:
can we rename the controller to `admin-flash` instead? I can see us using a more generic `flash` controller in the future for animations etc on all flashes across the site.